biggraphite/drivers/cassandra.py

```diff
@@ -208,14 +238,28 @@ class InvalidGlobError(InvalidArgumentError):
 
 # CONSISTENCY PARAMETERS
 # ======================
+
+CONSISTENCIES = {
```

Comment:
You can use https://github.com/datastax/python-driver/blob/5c5a53ef8de50a1da48964e9bc8e196f3780ec4b/cassandra/__init__.py#L95